components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/profiler/Profiler.kt

```diff
@@ -63,4 +64,21 @@ class Profiler(
     override fun addMarker(markerName: String) {
         runtime.profilerController.addMarker(markerName)
     }
+
+    override fun startProfiler(filters: Array<String>, features: Array<String>) {
+        runtime.profilerController.startProfiler(filters,features)
```

Comment:
chore (blocking): I think you'll need to run ktlint here. There should be a space after the comma.